qmcpy/discrete_distribution/lattice/lattice.py

```diff
@@ -123,6 +155,8 @@ def __init__(self, dimension=1, randomize=True, order='natural', seed=None,
         self.mimics = 'StdUniform'
         self.low_discrepancy = True
         super(Lattice,self).__init__(dimension,seed)
+        if isinstance(generating_vector,int):
+            self.z_og = append(uint64(1),2*self.rng.integers(1,2**(self.m_max-2),size = dimension-1,dtype = uint64)+1)
```

Comment:
remove spaces before and after `=`

I think it should be 
`2*self.rng.integers(1,2**(self.m_max-1),size = dimension-1,dtype = uint64)+1`
so values are in 
$$[2(1)+1,2(2^{m-1}-1)+1] = [3,2^m-1].$$
The current line
`2*self.rng.integers(1,2**(self.m_max-2),size = dimension-1,dtype = uint64)+1`
produces values in 
$$[2(1)+1,2(2^{m-2}-1)+1 = [3,2^{m-1}-1].$$